src/Mount.cpp

```diff
@@ -772,8 +772,8 @@ void Mount::configureFocusDriver(Stream *serial, float rsense, byte driveraddres
         #if USE_VREF == 0  //By default, Vref is ignored when using UART to specify rms current.
     _driverFocus->I_scale_analog(false);
         #endif
-    LOGV2(DEBUG_STEPPERS, F("Mount: Requested Focus motor rms_current: %d mA"), rmscurrent);
-    _driverFocus->rms_current(rmscurrent, 1.0f);  //holdMultiplier = 1 to set ihold = irun
+    LOGV2(DEBUG_STEPPERS | DEBUG_FOCUS, F("Mount: Requested Focus motor rms_current: %d mA"), rmscurrent);
+    _driverFocus->rms_current(rmscurrent, FOCUSER_MOTOR_HOLD_SETTING / 100f);  //holdMultiplier = 1 to set ihold = irun
```

Comment:
`100f` is an invalid specifier? Does this code get checked by `matrix_build.py`? If not we need to add it

Reply:
The rms_current input is supposed to be a float value between 0-1. I assume this should be 100.0f rather than 100f?

Reply:
Well, well, well, TIL. The C spec does require a . for floating point constants.... I guess some compilers are a little more lenient.

Reply:
Could you apply this patch to make `matrix_build.py` test focus builds?
[add_focus_to_matrix_build.txt](https://github.com/OpenAstroTech/OpenAstroTracker-Firmware/files/7118501/add_focus_to_matrix_build.txt)
(`git apply add_focus_to_matrix_build.txt`)
@andre-stefanov can you check that I modified the build script correctly? I'm not 100% sure it's correct

Reply:
avr-g++ (AVR_8_bit_GNU_Toolchain_3.6.2_1759) 5.4.0